pkgs/top-level/python-packages.nix

```diff
@@ -22252,6 +22252,27 @@ in {
   };
 
 
+  rst2html5 = buildPythonPackage rec {
+    name = "rst2html5-${version}";
+    version = "1.8.2";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/bc/33/86da22a1b2fc63403f19567d9e07be311881abeb3e0380d7f6d7faf8b8b6/${name}.tar.gz";
```

Comment:
please don't use an url like this because it has to be changed each time when the version changes.